ql/src/test/results/clientpositive/llap/subquery_multi.q.out

```diff
@@ -246,7 +246,7 @@ POSTHOOK: Input: default@part_null
 85768	almond antique chartreuse lavender yellow	Manufacturer#1	Brand#12	LARGE BRUSHED STEEL	34	SM BAG	1753.76	refull
 86428	almond aquamarine burnished black steel	Manufacturer#1	Brand#12	STANDARD ANODIZED STEEL	28	WRAP BAG	1414.42	arefully 
 90681	almond antique chartreuse khaki white	Manufacturer#3	Brand#31	MEDIUM BURNISHED TIN	17	SM CASE	1671.68	are slyly after the sl
-Warning: Shuffle Join MERGEJOIN[57][tables = [$hdt$_0, $hdt$_1, $hdt$_2, $hdt$_3]] in Stage 'Reducer 4' is a cross product
+Warning: Shuffle Join MERGEJOIN[57][tables = [$hdt$_0, $hdt$_2, $hdt$_3]] in Stage 'Reducer 4' is a cross product
```

Comment:
The `$hdt$_1` is omitted because `$hdt$_0` and `$hdt$_1` perform a left semi join, that is `p_name IN (select p_name from part_null)`  in `Reducer 2`, the `$hdt$_1` does not produces output after this [join](https://github.com/apache/hive/blob/23b0d30051af3ade5b03496ab9cf22929168dc84/ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java#L9109-L9118), only `$hdt$_0`, `$hdt$_2`(distinct p_brand) and `$hdt$_2`(count p_name) take place in `$hdt$_4`.